startauth.sh.example

```diff
@@ -0,0 +1,11 @@
+# at all times have only one line uncommented
+# PROXY_PREFIX value defines the name of the other 2 environmnet variables
+
+# example to proxy to staging
+PROXY_PREFIX=STAGING STAGING_URL=https://nxt3.staging.lizard.net/ STAGING_API_KEY=123456789STAGINGKEY yarn start
+
+# example to proxy to production
+# PROXY_PREFIX=LIZARD LIZARD_URL=https://demo.lizard.net/ LIZARD_API_KEY=123456789PRODUCTIONKEY yarn start
+
```

Comment:
Misschien moet de default prefix geen "STAGING" zijn, maar "PROXY" ?

Want op een command line een PREFIX meegeven is heel gek, je kunt ook gewoon de default variabele namen gebruiken dan. Maar `STAGING_URL=https://demo.lizard.net/` staat ook gek.

Als we de default prefix PROXY maken kan dit gewoon

`PROXY_URL=https://demo.lizard.net/ PROXY_API_KEY=123345667467 yarn start`

zijn.

PREFIX heeft alleen zin in de use case waar je een config file hebt met meerdere verschillende settings er in. Als je zoals hier het systeem "always leave one line uncommented" gebruikt dan zou je geen prefix moeten gebruiken, dat is dubbel.